pkg/exporter/exporter.go

```diff
@@ -164,6 +180,12 @@ func New(server string, timeout time.Duration, logger log.Logger) *Exporter {
 			nil,
 			nil,
 		),
+		rejectedConnections: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, "", "rejected_connections"),
```

Comment:
If this is a counter, it should end in `_total`.
```suggestion
			prometheus.BuildFQName(Namespace, "", "rejected_connections_total"),
```

Reply:
Yeah, this is a count.